OrbitQt/ConnectionArtifacts.h

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_QT_CONNECTION_ARTIFACTS_H_
+#define ORBIT_QT_CONNECTION_ARTIFACTS_H_
+
+#include <memory>
+
+#include "DeploymentConfigurations.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitClientData/ProcessData.h"
+#include "OrbitClientServices/ProcessManager.h"
+#include "OrbitGgp/Instance.h"
+#include "OrbitSsh/Context.h"
+#include "OrbitSsh/Credentials.h"
+#include "grpcpp/channel.h"
+#include "servicedeploymanager.h"
+
+namespace orbit_qt {
+
+// A class to store remote connection related artifacts. This is used by the ProfilingTargetDialog
+// and OrbitApp to be able to use the same connection. The
```

Comment:
The...?

Reply:
I rewrote ConnectionArtifacts.h which is now ConnectionConfiguration.h. While I did that, I removed the comment, because I felt the code is self-explaining enough. Let me know if you think otherwise.

(Ps. the The was a mistake)